include/sensors/core/mmr920C04.hpp

```diff
@@ -0,0 +1,275 @@
+#pragma once
```

Comment:
You mentioned in your PR comment that you were worried about putting some of this functionality here. I kind of agree with that assessment, but honestly I think it's about the naming. This does what `capacitive_sensor_callbacks.hpp` does, not what `hdcwhatever` does. I think a file named like this should just have conversion and utility functions with little state, and have something that's more closely associated with the task to handle the actual data routing. So moving this to `sensors/tasks` and putting the actual conversion functions back into `sensors/core` seems perfect.